pkg/apis/serving/v1alpha1/framework_tensorflow.go

```diff
@@ -58,7 +63,7 @@ func (t *TensorflowSpec) CreateModelServingContainer(modelName string, config *F
 			"--port=" + TensorflowServingGRPCPort,
 			"--rest_api_port=" + TensorflowServingRestPort,
 			"--model_name=" + modelName,
-			"--model_base_path=" + t.ModelURI,
+			"--model_base_path=" + constants.DefaultModelLocalMountPath,
```

Comment:
Does this force tensorflow serving to use model initializer ?

Reply:
It does yes.